.github/workflows/freethreading_tests.yml

```
@@ -6,7 +6,7 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install uv and set the python version
-        uses: astral-sh/setup-uv@v5
+        uses: astral-sh/setup-uv@v6
         with:
           version: "0.7.2"
```

```suggestion
          version: "latest"
```

@HaoZeke @rgommers was there a reason the uv version was pinned like this?